src/Libraries/LibAsset.sol

```diff
@@ -77,7 +77,7 @@ library LibAsset {
         address assetId,
         address recipient,
         uint256 amount
-    ) private {
+    ) internal {
```

Comment:
This requires a version change. 
Also please I would suggest to then make `transferNativeAsset` internal as well